src/librustc_codegen_ironox/lib.rs

```diff
@@ -163,7 +170,7 @@ impl ModuleIronOx {
     }
 
     pub fn asm(&self) -> String {
-        "not actual asm".to_string()
+        "nop".to_string()
```

Comment:
This is just any old asm to be populated later?

Reply:
Yep, this is the function that translates the instructions in the module to asm.

Reply:
OK